prov/efa/src/rdm/efa_rdm_ep.h

```diff
@@ -59,6 +59,7 @@ struct rxr_queued_copy {
 };
 
 #define RXR_EP_MAX_QUEUED_COPY (8)
+#define EFA_RDM_EP_MAX_WR_PER_IBV_POST_SEND (4096)
```

Comment:
How is this magic number derived? some experiments?

Reply:
This is the value of `efa_max_outstanding_tx_ops`, which was derived from device.

Reply:
I will added a comment.

Reply:
added a comment in new revision.